src/loaders/external_png/tvgPngLoader.cpp

```diff
@@ -42,6 +42,24 @@ static void _premultiply(uint32_t* data, uint32_t w, uint32_t h)
 }
 
 
+static inline uint32_t CHANGE_COLORSPACE(uint32_t c)
```

Comment:
We don't need this duplicate logic if you consider the general Converter mentioned above.